app/controllers/predicates/AuthoriseAsAgentWithClient.scala

```diff
@@ -27,48 +27,55 @@ import services.EnrolmentsAuthService
 import uk.gov.hmrc.auth.core._
 import uk.gov.hmrc.auth.core.retrieve.{Retrievals, ~}
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import views.html.errors.standardError
 
 import scala.concurrent.Future
 
 @Singleton
 class AuthoriseAsAgentWithClient @Inject()(enrolmentsAuthService: EnrolmentsAuthService,
                                            val errorHandler: ErrorHandler,
                                            implicit val messagesApi: MessagesApi,
-                                           implicit val appConfig: AppConfig)
-  extends FrontendController with AuthBasePredicate with I18nSupport with ActionBuilder[User] with ActionFunction[Request, User] {
+                                           implicit val appConfig: AppConfig
+                                           )
+  extends FrontendController with AuthBasePredicate with I18nSupport with ActionBuilder[User]
```

Comment:
This file is staying in so the test spec is required for it. Can you re-add it please